resources/views/wiki/_sitemap_section.blade.php

```diff
@@ -0,0 +1,38 @@
+{{--
+    Copyright (c) ppy Pty Ltd <contact@ppy.sh>.
+
+    This file is part of osu!web. osu!web is distributed with the hope of
+    attracting more community contributions to the core ecosystem of osu!.
+
+    osu!web is free software: you can redistribute it and/or modify
+    it under the terms of the Affero GNU General Public License version 3
+    as published by the Free Software Foundation.
+
+    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+    See the GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+--}}
+
+@php
+    $page = is_array($sitemap) ? $parent : $sitemap;
```

Comment:
Only use `$parent` as the value will always be the same?

```suggestion
    $page = $parent;
```

`$page` can probably even be removed. Or rename `$parent` to `$page` as it took me a while wondering why link for `$parent` is generated here instead of the caller (parent).

(and [as I said before](https://github.com/ppy/osu-web/pull/5652#discussion_r390757197), I don't see point filling [the value with `$page->path` instead of just null](https://github.com/ppy/osu-web/pull/5652/files#diff-897db912bb97ca161df702078958e918R111) as they may be overwritten by array anyway)